bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/StaticReferenceRewriteTransformerFactory.java

```diff
@@ -106,6 +107,9 @@ public void startElement(String namespaceURI, String localName, String qName, At
             description = "Path prefixes to rewrite.")
     private static final String PROP_PREFIXES = "prefixes";
 
+    @Property(label = "Override existing host", description = "This property allows you to override the existing host in the attribute that has to be rewritten")
```

Comment:
Please add a default (of false) here so this gets recognized as a `boolean` property.